projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -49,11 +38,46 @@
 			} );
 		}
 
+		function getAverageColor( imgEl ) {
+			var canvas = document.createElement( 'canvas' ),
+				context = canvas.getContext && canvas.getContext( '2d' ),
+				imgData,
+				width,
+				height,
+				length,
+				rgb = { r: 0, g: 0, b: 0 },
+				count = 0;
+
+			if ( ! imgEl ) {
+				return rgb;
+			}
+
+			height = canvas.height = imgEl.naturalHeight || imgEl.offsetHeight || imgEl.height;
+			width = canvas.width = imgEl.naturalWidth || imgEl.offsetWidth || imgEl.width;
+
+			context.drawImage( imgEl, 0, 0 );
+			imgData = context.getImageData( 0, 0, width, height );
```

Comment:
Do we need a `imgEl.crossOrigin = "Anonymous";` above this line?

I'm seeing `'CanvasRenderingContext2D': The canvas has been tainted by cross-origin data.` in my testing (desktop)